cmd/root.go

```diff
@@ -130,4 +140,7 @@ func init() {
 	rootCmd.Flags().DurationVar(&rootCmdOpts.watchAvailableStorageInterval, "watch-storage-available-size-interval", 5*time.Second, "Interval to check if the disk is running low on space. Set to 0 to disable the periodic disk size check")
 	rootCmd.Flags().Uint64Var(&rootCmdOpts.watchAvailableStorageMinBytes, "watch-storage-available-size-min-bytes", 10*1024*1024, "Minimum required available disk size (in bytes) to continue operation. If available disk space gets below this threshold, then the --low-available-storage-action is performed")
 	rootCmd.Flags().StringVar(&rootCmdOpts.lowAvailableStorageAction, "low-available-storage-action", "none", "Action to perform in case the available storage is low. One of (none|handover|terminate). none means no action is performed. handover means the dqlite node will handover its leadership role, if any. terminate means this dqlite node will shutdown")
+	rootCmd.Flags().StringVar(&rootCmdOpts.admissionControlPolicy, "admission-control-policy", "allow-all", "Policy to use for the admission control ('allow-all'|'limit'). Disabled by default (='allow-all')")
+	// TODO(MK-1398): Find a sane default value here by observing normal/high load in microk8s
+	rootCmd.Flags().Int64Var(&rootCmdOpts.acpLimitMaxRunningTxn, "admission-control-policy-limit-max-running-txn", 300, "Max number of concurrently running transactions. Txns above this threshold will be denied.")
```

Comment:
This is user-facing text, let's make sure it properly explains the context.
```suggestion
	rootCmd.Flags().Int64Var(&rootCmdOpts.acpLimitMaxRunningTxn, "admission-control-policy-limit-max-concurrent-transactions", 300, "Maximum number of transactions that are allowed to run concurrently. Transactions will not be admitted after the limit is reached.")
```